inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -28,6 +28,7 @@
 
 from refextract import extract_journal_reference, extract_references_from_file
 
+from inspirehep.config import REFEXTRACT_JOURNAL_KB_PATH
```

Comment:
The bottleneck is how many journals will be added. I think now is tested with 3500 journals more or less but the time is quite small anyway I think.